include/picongpu/plugins/adios/ADIOSWriter.hpp

```diff
@@ -945,8 +950,10 @@ class ADIOSWriter : public IIOBackend
         std::stringstream strFname;
         strFname << restartFilename << "_" << mThreadParams.currentStep << ".bp";
 
+        const std::string filename = strFname.str( );
+
         // adios_read_open( fname, method, comm, lock_mode, timeout_sec )
-        log<picLog::INPUT_OUTPUT > ("ADIOS: open file: %1%") % strFname.str();
+        log<picLog::INPUT_OUTPUT > ("ADIOS: open file: %1%") % filename;
```

Comment:
please add proper spacing while editing this line :)
```suggestion
        log< picLog::INPUT_OUTPUT > ("ADIOS: open file: %1%") % filename;
```

Reply:
Done